packages/headless/src/controllers/commerce/field-suggestions/headless-category-field-suggestions.ts

```diff
@@ -83,7 +84,7 @@ export function buildCategoryFieldSuggestions(
 
   const facetSearch = buildCategoryFacetSearch(engine, {
     options: {
-      facetId: options.facetId,
+      facetId: namespaceCommerceFieldSuggestionFacet(options.facetId),
```

Comment:
You could set the value in a variable and reuse it instead of calling namespaceCommerceFieldSuggestionFacet twice